tests/integration/api/detection/test_api_detection.py

```diff
@@ -130,6 +122,20 @@ def setup_configurable_parameters(template_dir, num_iters=10):
         hyper_parameters.postprocessing.confidence_threshold = 0.1
         return hyper_parameters, model_template
 
+
+class TestMPADetAPI(MPADetAPIBase):
```

Comment:
It was renamed from `TestMPADetAPI` to `TestDetectionTaskAPI` for some reason. Let's revert to the new name? 
The base class can be called `DetectionTaskAPIBase`.

Reply:
Yes, thank you. Same as classification?

Reply:
let's align cls and det for consistency?